### DISCUSSION TITLE
Vaccination campaign metadata (Canton ZH)

### DISCUSSION THREAD

**Discussion by andreasamsler** [openZH/covid_19#1663] [General]
Please give your feedback / ask your questions to https://github.com/openZH/covid_19/tree/master#canton-zurich-vaccination-campaign in a comment. Many thanks, best aa

**Comment by rsalzer** [comment]
I really like the detailed information.
Just for clarification: All the variables are for one record - so one record is for one person, including 1st and 2nd dose and gets updated once a person gets their 2nd dose?

**Reply by andreasamsler** [reply]
Thx, I assume that's the case. We're claryifing with GD-ZH. Keep you posted.

**Reply by andreasamsler** [reply]
Status update (2021-02-19):
- GD-ZH: response pending; we are waiting for feedback
- BAG: feedback is sought from relevant units and communicated to us asap
